apps/cms/scaffold/settings.php.append.txt

```diff
@@ -14,7 +14,7 @@ $config['silverback_external_preview.settings'] = [
 
 $config['graphql.graphql_servers.main']['schema_configuration']['directable']['build_webhook'] = $publisherUrl . '/___status/build';
 $config['graphql.graphql_servers.main']['schema_configuration']['directable']['update_webhook'] = $publisherUrl . '/___status/update';
-$config['graphql.graphql_servers.main']['schema_configuration']['directable']['build_url'] = $publisherUrl;
+$config['graphql.graphql_servers.main']['schema_configuration']['directable']['build_url'] = $netlifyUrl;
```

Comment:
@colorfield Is `build_url` this used on local? If yes, we might need a condition here.

Reply:
thanks for catching @Leksat it's the main settings and we don't have local override so adding the condition.

Reply:
As long as no env var override is set locally, then it wouldn't cause issue due to https://github.com/AmazeeLabs/silverback-template/blob/release/apps/cms/scaffold/settings.php.append.txt#L6-L7, that's probably why tests were green, but we shouldn't assume this so checking it here https://github.com/AmazeeLabs/silverback-template/pull/325/commits/1c98c0cbaba4631750b2f83f3bd22590a8ac2eb1